cmd/spire-server/cli/agent/list.go

```diff
@@ -1,104 +1,86 @@
 package agent
 
 import (
-	"errors"
 	"flag"
 	"fmt"
-	"os"
 	"time"
 
+	"github.com/mitchellh/cli"
+
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
 	"github.com/spiffe/spire/cmd/spire-server/util"
-	"github.com/spiffe/spire/proto/spire/api/registration"
-	"github.com/spiffe/spire/proto/spire/common"
+	common_cli "github.com/spiffe/spire/pkg/common/cli"
+	"github.com/spiffe/spire/proto/spire/api/server/agent/v1"
+	"github.com/spiffe/spire/proto/spire/types"
 
 	"golang.org/x/net/context"
 )
 
-//ListConfig holds configuration for ListCLI
-type ListConfig struct {
-	// Socket path of registration API
-	RegistrationUDSPath string
+type listCommand struct {
+	agents []*types.Agent
```

Comment:
why is this being stored here?

Reply:
No good reason for that. I will remove it. Good catch!